dotnet/aspnetcore-runtime-2.1/Portfile

```diff
@@ -34,15 +37,19 @@ extract.mkdir       yes
 
 use_configure       no
 
-depends_run         port:dotnet-cli
+depends_run         port:dotnet-cli \
+                    port:dotnet-runtime-2.1
```

Comment:
@silverbzez I initially thought about this design, but feel that it requires synchronizing aspnetcore-runtime/dotnet-runtime versions. My lack of expertise in both dotnet and macports didn't allow me to come up with a better solution than to have aspnetcore/dotnet as independent ports. BTW, that was what I also observed when installing them using vendor packages.

Reply:
From what I have tested, `dotnet-runtime` is a must-required dependency for `aspnetcore-runtime` as `dotnet-runtime` provides standard libraries like io, data structures, reflections, etc. that are definitely needed for a complicated aspnet project. They can't be independent ports anyway.
However, both versions can be unsynced as long as they are of the same major version. Most of the updates are just sercurity fixes that are guaranteed ABI compatible.